src/ResourceManager/DataFactories/Commands.DataFactories/Pipelines/SuspendAzureDataFactoryPipelineCommand.cs

```diff
@@ -40,12 +43,6 @@ public override void ExecuteCmdlet()
             }
 
             ConfirmAction(
-                Force.IsPresent,
-                string.Format(
```

Comment:
@bgold09 Correct, unless ```ConfirmPreference``` is < Medium, or the user specifies -Confirm, in which case the ShouldProcess confirmation is shown.